content/en/docs/Tutorials/tempo/02-kiali-tempo-integration.md

```diff
@@ -0,0 +1,244 @@
+  accessModes:
+    - ReadWriteOnce
+  resources:
+    # This is the request for storage. Should be available in the cluster.
+    requests:
+      storage: 5Gi
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: minio
+spec:
+  selector:
+    matchLabels:
+      app: minio
+  strategy:
+    type: Recreate
+  template:
+    metadata:
+      labels:
+        # Label is used as selector in the service.
+        app: minio
+    spec:
+      # Refer to the PVC created earlier
+      volumes:
+        - name: storage
+          persistentVolumeClaim:
+            # Name of the PVC created earlier
+            claimName: minio-pv-claim
+      initContainers:
+        - name: create-buckets
+          image: busybox:1.28
+          command:
+            - "sh"
+            - "-c"
+            - "mkdir -p /storage/tempo-data"
+          volumeMounts:
+            - name: storage # must match the volume name, above
+              mountPath: "/storage"
+      containers:
+        - name: minio
+          # Pulls the default Minio image from Docker Hub
+          image: minio/minio:latest
+          args:
+            - server
+            - /storage
+            - --console-address
+            - ":9001"
+          env:
+            # Minio access key and secret key
+            - name: MINIO_ACCESS_KEY
+              value: "minio"
+            - name: MINIO_SECRET_KEY
+              value: "minio123"
+          ports:
+            - containerPort: 9000
+            - containerPort: 9001
+          volumeMounts:
+            - name: storage # must match the volume name, above
+              mountPath: "/storage"
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: minio
+spec:
+  type: ClusterIP
+  ports:
+    - port: 9000
+      targetPort: 9000
+      protocol: TCP
+      name: api
+    - port: 9001
+      targetPort: 9001
+      protocol: TCP
+      name: console
+  selector:
+    app: minio
+```
+
+And apply yaml: 
+```
+kubectl apply -n tempo -f minio.yaml
+```
+
+We will create a secret to access minio:
+```
+kubectl create secret generic -n tempo tempostack-dev-minio \
+--from-literal=bucket="tempo-data" \
+--from-literal=endpoint="http://minio:9000" \
+--from-literal=access_key_id="minio" \
+--from-literal=access_key_secret="minio123"
+```
+
+Install Grafana tempo with the operator. We will use the secret created in the previous step:
+```
+kubectl apply -n tempo -f - <<EOF
+apiVersion: tempo.grafana.com/v1alpha1
+kind: TempoStack
+metadata:
+  name: smm
+spec:
+  storageSize: 1Gi
+  storage:
+    secret:
+      type: s3
+      name: tempostack-dev-minio
+  resources:
+    total:
+      limits:
+        memory: 2Gi
+        cpu: 2000m
+  template:
+    queryFrontend:
+      jaegerQuery:
+        enabled: true
+        ingress:
+          type: ingress
+EOF
+```
+
+(Optional) We can check if all the deployments have started correctly, and the services distributor has the port 9411 and the query frontend 16686:
```

Comment:
Here we can say "It's not required" or "as an optional step"

Reply:
Ok, updated.